modules/social_features/social_profile/social_profile.info.yml

```diff
@@ -20,4 +20,5 @@ dependencies:
   - drupal:user
   - drupal:views
   - select2:select2
+  - config_modify:config_modify
```

Comment:
I think it makes sense to have these in the separate features / custom modules info file instead of the entire profile. As not all of Open Social is requiring this and that way we can easily separate concerns or if we phase out modules we don't have dependencies left behind.

Reply:
My bad, this was perfectly fine! I thought it was the social distro info yml